core/Cargo.toml

```diff
@@ -51,15 +53,15 @@ pin-project = "1.0"
 prometheus = "0.13"
 prost = "0.11"
 prost-types = { version = "0.4", package = "prost-wkt-types" }
+reqwest = { version = "0.11", features = ["json", "stream", "rustls-tls", "tokio-rustls"], default-features = false, optional = true }
 rand = "0.8.3"
-reqwest = { version = "0.11", features = ["json", "stream", "rustls-tls", "tokio-rustls"], default-features = false }
```

Comment:
Alphabetical order got messed up here

Reply:
There are other ordering mismatches as well.
Should I fix them as well using 'cargo sort' ?

Reply:
Sure, sgtm